core/src/i18n/en.yaml

```
@@ -222,10 +231,27 @@ clusters:
       unknown: Unknown storage type
     labels:
       inMemory: In memory
-      localStorage: localStorage
-      sessionStorage: sessionStorage
+      localStorage: Local storage
+      sessionStorage: Session storage
       unknown: Unknown
     title: Storage Type
+  wizard:
+    context: Current context
+    kubeconfig: Provide Kubeconfig
+    update: Provide Authentication
+    storage: Privacy Options
+    intro: To add a cluster, all you have to do is provide the cluster configuration – called “kubeconfig” in the Kubernetes world.
+    storage-info: |
+      Don't worry, we don't store your kubeconfig file on the server. The information stays in your browser.
+      You can choose your privacy options in the last step.
+    incomplete: We couldn't find enough authentication information in your kubeconfig file. You can enter it manually.
+    editor-label: 'or paste your kubeconfig here:'
+    token-info: Consult your authorization provider for information how to aquire the token.
```

```suggestion
    token-info: If you don't know how to get your token, ask your authentication provider.
```

I believe it's "authentication" not "authorization" here, but correct me if I'm mistaken :)

---

you're absolutely right